wiki/Tournaments/Official_support/en.md

```diff
@@ -124,7 +124,8 @@ Registrant filtering, not to be confused with [screening](#tournament-screening)
 
 In addition to the expectations for programs above, tools or systems used to filter registrants or seed players must also abide by the following practices:
 
-- If a dataset is used, it must be up to date, publicly available and downloadable. Documentation must be provided explaining how to recreate the dataset and reproduce the results using it — including code where applicable.
+- If a dataset of osu! multiplayer matches (or otherwise) is used, archives must be publicly available, downloadable, timestamped, and have the most recent archive be no more than 30 days out of date. Archives must remain available for at least 6 months. Documentation must be provided explaining how to reproduce the results using it — including code where applicable.
```

Comment:
```suggestion
- If a dataset of osu! multiplayer matches, player scores or otherwise is used, archives must be publicly available, downloadable, timestamped, and have the most recent archive be no more than 30 days out of date. Archives must remain available for at least 6 months. Documentation must be provided explaining how to reproduce the results using it — including code where applicable.
```
Though I recognise "or otherwise" is present here as a catch all, I would also like to see player scores mentioend, since  multiplayer matches and user scores combined cover 98% of cases.

Reply:
Would a `,` be needed before `or`?

Reply:
Applied